addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -86,6 +86,13 @@ def __init__(self, identifier, device):
         })
 
         self.receipt_protocol = 'star' if 'STR_T' in device['device-id'] else 'escpos'
+
+        if any(cmd in device['device-id'] for cmd in ['CMD:STAR;', 'CMD:ESC/POS;']):
+            self.device_subtype = "receipt_printer"
+        elif "COMMAND SET:ZPL;" in device['device-id']:
+            self.device_subtype = "label_printer"
+        else:
+            self.device_subtype = "office_printer"
```

Comment:
Same as below for windows, "office_printer" doesn't correspond to the "other_printer" defined in your model "subtype" selection field